packages/fxa-auth-server/docs/swagger/totp-api.ts

```diff
@@ -154,9 +178,22 @@ const TOTP_SETUP_COMPLETE_POST = {
   ],
 };
 
+const MFA_TOTP_SETUP_COMPLETE_POST = {
+  ...TAGS_TOTP,
+  description: '/mfa/totp/setup/complete',
+  notes: [
+    dedent`
+      🔒 Authenticated with MFA JWT (scope: mfa:2fa)
```

Comment:
I like that you documented the required scope. I might have to go back and update some of my docs. 👍